stackit/services/mariadb/instance/datasource.go

```diff
@@ -174,8 +177,33 @@ func (r *instanceDataSource) Read(ctx context.Context, req datasource.ReadReques
 		core.LogAndAddError(ctx, &resp.Diagnostics, "Mapping fields", err.Error())
 		return
 	}
+
+	// Compute and store values not present in the API response
+	r.loadPlanNameAndVersion(ctx, &diags, &state)
+
 	// Set refreshed state
 	diags = resp.State.Set(ctx, &state)
 	resp.Diagnostics.Append(diags...)
 	tflog.Info(ctx, "MariaDB instance read")
 }
+
+func (r *instanceDataSource) loadPlanNameAndVersion(ctx context.Context, diags *diag.Diagnostics, model *Model) {
```

Comment:
see comment below, but I think here we should not duplicate the function and use the one defined in `resource.go`, like we do for the `mapFields` functions